### PR TITLE
Remove FireEventIfNotNull

### PR DESCRIPTION
**Pull request by Bond-009**
It's a pretty useless "helper" class

### CODE REVIEW

File: MediaBrowser.Common/Events/EventHelper.cs [hunk 1]
```diff
@@ -59,50 +60,5 @@ public static void QueueEventIfNotNull<T>(EventHandler<T> handler, object sender
...
-        {
-            if (handler != null)
-            {
-                try
-                {
-                    handler(sender, args);
-                }
-                catch (Exception ex)
-                {
-                    logger.ErrorException("Error in event handler", ex);
```
Comment (JustAMan) [inline MediaBrowser.Common/Events/EventHelper.cs]: Your change seem to be changing the logic, as previously an error in the handler was logged but not propagated further.

Are you sure that your fix is fully correct?

Comment (Bond-009) [reply MediaBrowser.Common/Events/EventHelper.cs]: Wrapping a catch around something isn't a proper fix, not having events throw exceptions is.

Comment (JustAMan) [reply MediaBrowser.Common/Events/EventHelper.cs]: I agree, in theory. In practice I just want to have progress that is not breaking user experience :)

Comment (JustAMan) [reply MediaBrowser.Common/Events/EventHelper.cs]: @Bond-009 no response... though I think I can live with the logic change as it should eventually make Jellyfin better - more errors **not** silently eaten means we can faster clean them up.

**Pull request merged**